docs/src/usage.md

```diff
@@ -50,6 +56,16 @@ The session token shoud be passed as an environment variable `AWS_SESSION_TOKEN`
 
 To check if IAM role assumption is needed run `aws s3 ls s3://your-bucket-name --region your-region`. If you get a `403 Forbidden` error, you might need an assumed role
 
+##### Streaming from Google cloud storage
+
+To load tensors from GCS pass credentials as environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
+
+The streamer supports GCS url `gs://my-bucket/my/file/path.safetensors`
+
+> **Note:** If using GCS url with prefix `gs://` there is no need to pass additional environment variables for S3 compatible storage 
+
+> **Note for vLLM users:** GCS url not supported in vLLM - if using the streamer in vLLM follow instructions for S3 compatible storage
```

Comment:
This line is a bit not in place,
First, why not making it work in vLLM?

Second, I think it should be part of vLLM docs